src/components/CustomIcon.vue

```diff
@@ -1,137 +1,40 @@
 <template>
-  <component :is="url" :class="className" />
+  <customIcon :class="className" />
```

Comment:
Icons in socials component aren't working because in [Community.vue](https://github.com/gabaldon/sheikah/blob/feat/vue3/src/components/Community.vue#L37) the img name should be in lowercase